File: backend/src/main/java/cohappy/backend/model/dto/response/GetHouseDTO.java
package cohappy.backend.model.dto.response;

import lombok.Data;

import java.util.List;

@Data
public class GetHouseDTO {
    private String houseCode;
    private List<String> admins;
    private List<String> users;
    private List<byte[]> images;
    private int costPerMonth;
    private String country;
    private String region;
    private String street;
    private int civicNumber;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/response/GetHouseDTO.kt
package cohappy.frontend.model.dto.response;
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class GetHouseDTO(
    var houseCode: String? = null,
    var admins: List<String>? = null,
    var users: List<String>? = null,
    var images: List<ByteArray>? = null,
    var costPerMonth: Int? = null,
    var country: String? = null,
    var region: String? = null,
    var street: String? = null,
    var civicNumber: Int? = null
)